apps/dg/resources/dg.css

```diff
@@ -42,6 +42,30 @@
 @font-face {font-family: 'icomoon';src: static_url('icon_fonts/icomoon.woff') format('woff');}
 @font-face {font-family: 'icomoon';src: static_url('icon_fonts/icomoon.svg') format('svg');}
 
+html, body {
+    /* cf. https://www.bram.us/2016/05/02/prevent-overscroll-bounce-in-ios-mobilesafari-pure-css/ */
+    position: fixed;
+    overflow: hidden;
+    /* cf. https://developers.google.com/web/updates/2017/11/overscroll-behavior */
+    overscroll-behavior: none;
+
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-touch-callout: none;
+}
+
+/* SproutCore inexplicably left out -ms-user-select and -webkit-user-select */
+.sc-view.allow-select {
+    -khtml-user-select: text;
+    -moz-user-select: text;
+    -ms-user-select: none;
```

Comment:
Why "none" for ms-user-select? Do we not want to allow selection there?

Reply:
Good catch. Copy/paste error.